src/format.h

```diff
@@ -92,6 +92,8 @@ extern const struct format_loader libxmp_loader_stc;
 #define NUM_FORMATS 3
 #endif
 
+LIBXMP_BEGIN_DECLS
+
```

Comment:
Function-only, but not data? (There are possibly other similar places, so, same for them too.)

Reply:
Not sure what I was thinking there, will fix in a little bit.